pkg/apis/serving/v1beta1/revision_validation_test.go

```diff
@@ -572,3 +573,91 @@ func TestImmutableFields(t *testing.T) {
 		})
 	}
 }
+
+func TestRevisionTemplateSpecValidation(t *testing.T) {
```

Comment:
Interesting that this was missing all along. Any reason why you didn't copy all of the tests over?

Reply:
I dint copy test case related to `serving.QueueSideCarResourcePercentageAnnotation` 
because in v1beta/revision_validation.go we dont have this logic
https://github.com/knative/serving/blob/master/pkg/apis/serving/v1alpha1/revision_validation.go#L87 so test case related to this i have skipped.

I wanted to know like why we have not added https://github.com/knative/serving/blob/master/pkg/apis/serving/v1alpha1/revision_validation.go#L87  in v1beta1
Any reason?

Reply:
This seems like a bug. We should not have difference in annotations here between the two versions.

Reply:
I submitted a bug to track this and keep this PR scoped to just this feature. https://github.com/knative/serving/issues/4903